### ISSUE TITLE
Carousel View Focus

### ISSUE THREAD

**Issue by Malikkhoja** [AndreiMisiukevich/CardView#107]
I have a requirement of putting the carousel view in ScrollView. However that works fine for me. But I am not able to scroll my page when my focus is on Carousel View. 

Any help would be great.

Thanks in advance.

**Comment by Malikkhoja** [comment]
Hi AndreiMisiukevich

Thanks for sharing the link.

It works great in Android.However there is no custom renderer for IOS. How do we handle this scenario in IOS?

Can you please help?

Thanks in Advance

**Comment by AndreiMisiukevich** [comment]
@Malikkhoja Hi, it works for iOS, but it isn't very pretty.

https://github.com/AndreiMisiukevich/CardView/blob/master/PanCardView/Controls/ParentScrollView.cs

This class handle iOS scenario.
Also you can try to set IsPanInteractionEnabled = false . (It will disable panGesture, but preserve possibility to swipe cards)

**Comment by AndreiMisiukevich** [comment]
This sample works well for me
https://github.com/AndreiMisiukevich/CardView/blob/master/PanCardViewSample/PanCardViewSample/Views/CarouselSampleViewNoTemplate.cs

**Comment by AndreiMisiukevich** [comment]
Just Set IsPanInteractionEnabled="False" in xaml
or CarouselView.IsPanInteractionEnabled=false in code

This prop belongs to CarouselView

**Comment by Malikkhoja** [comment]
Got it :)

However it doesnt work as expected. May be Swipe Gesture Recognizer we can try to play with IsPanInteractionEnabled value. Not sure how this would work.

**Comment by AndreiMisiukevich** [comment]
Why doesn't it work?
It should prevent pan gesture, but preserve swiping

**Comment by AndreiMisiukevich** [comment]
I open for any suggestions) how to fix this issue

**Comment by Malikkhoja** [comment]
The scrolling issue get solved if you add IsPanInteractionEnabled="False" however you cannot swipe different items of CarouselView. The swipe doesnt work. You cannot see the second image in the carousel.

**Comment by AndreiMisiukevich** [comment]
@Malikkhoja No, it prevents pan interaction, so you cannot move element, but you may swipe over card (fast move).

**Comment by Malikkhoja** [comment]
@AndreiMisiukevich  So can we change the element using swipe?

**Comment by AndreiMisiukevich** [comment]
@Malikkhoja Swipe -- yes
Pan -- no

if IsPanInteractionEnabled = false
